crates/ty_python_semantic/resources/mdtest/generics/pep695/variables.md

```diff
@@ -766,4 +766,23 @@ def constrained[T: (Callable[[], int], Callable[[], str])](f: T):
     reveal_type(f())  # revealed: int | str
 ```
 
+## Meta-type
+
+The meta-type of a typevar is the same as the meta-type of the upper bound, or the union of the
+meta-types of the constraints:
```

Comment:
Can we add the same stanza of tests to `legacy/variables.md` too?  (I don't think this particular code path would be effected, but we have seen tests that pass/fail inconsistently depending on which typevar flavor you use)